src/pages/storage/CustomVolumeCreateModal.tsx

```diff
@@ -62,7 +64,7 @@ const CustomVolumeCreateModal: FC<Props> = ({
           void queryClient.invalidateQueries({
             queryKey: [queryKeys.customVolumes],
           });
-          notify.success(`Storage volume ${values.name} created.`);
+          toastNotify.success(`Storage volume ${values.name} created.`);
```

Comment:
This doesn't work. The notification is displayed behind the modal, then the modal to select the newly created volume is shown and only after selection this modal closes and shows the notification. Maybe this is a candidate to keep an inline notification for the 2nd modal -- after the volume is created.

Reply:
Yeah I think inline is reasonable here since you can't really navigate anywhere while on this modal, unless if you close or cancel it

Reply:
great catch btw :+1: